build_tools/github_actions/runner/gcp/update_instance_groups.py

```diff
@@ -72,7 +72,7 @@ def get_migs(self, *, region, type, group, prefix, modifier=None):
     regions = [r.name for r in self._regions_client.list(request)]
 
     if type == "all":
-      type = r"\w+"
```

Comment:
Note that on its own this is going to match things past the type. I guess given that it's followed by the group match it will get appropriately limited

Reply:
Yeah, I was a little bit concerned about this, but `_` isn't allowed in the MIG name...

Reply:
Yeah this causes a problem: `--environment=prod --type=all --group=all` now matches testing groups. I think we should rename the new group type to just c16 or something